etc/kernel-launchers/bootstrap/bootstrap-kernel.sh

```diff
@@ -12,7 +12,7 @@ launch_python_kernel() {
     export JPY_PARENT_PID=$$  # Force reset of parent pid since we're detached
 
 	set -x
-	python ${KERNEL_LAUNCHERS_DIR}/python/scripts/launch_ipykernel.py --RemoteProcessProxy.kernel-id ${KERNEL_ID} --RemoteProcessProxy.response-address ${EG_RESPONSE_ADDRESS} --RemoteProcessProxy.spark-context-initialization-mode ${KERNEL_SPARK_CONTEXT_INIT_MODE}
+	python ${KERNEL_LAUNCHERS_DIR}/python/scripts/launch_ipykernel.py --RemoteProcessProxy.kernel-id ${KERNEL_ID} --RemoteProcessProxy.port-range ${EG_PORT_RANGE} --RemoteProcessProxy.response-address ${EG_RESPONSE_ADDRESS} --RemoteProcessProxy.spark-context-initialization-mode ${KERNEL_SPARK_CONTEXT_INIT_MODE}
```

Comment:
I think we should add an initializer at the top of file, something like:
```
EG_PORT_RANGE=${EG_PORT_RANGE:-0..0}
```
to prevent breakage in cases where the env isn't conveyed to the container - like for the docker process-proxies.  (wink, wink?)

Reply:
sounds like a solid idea

Reply:
done!